pkgs/applications/editors/jetbrains/source/build.nix

```
@@ -261,6 +245,9 @@ stdenvNoCC.mkDerivation rec {
     ../patches/bump-jackson-core-in-source.patch
   ];
 
+  # Regarding brokenPlugins.json:
+  #  We provide an empty brokenPlugins.json because the original file is 2.8 MB in size and can't be reliably
+  #  downloaded from archive.org without hash changes. The file is not important for the IDEs to function properly.
```

Not sure this needs to explicitly mention archive.org?


```suggestion
  #  downloaded without hash changes. The file is not important for the IDEs to function properly.
```